src/Orleans.Transactions.Testkit.Xunit/ConsistencyTransactionTestRunner.cs

```diff
@@ -1,25 +1,23 @@
-﻿using Orleans.Transactions.Tests.Consistency;
-using System;
+﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Threading;
+using System.Text;
 using System.Threading.Tasks;
+using Orleans.TestingHost;
+using Orleans.Transactions.TestKit.Base.Consistency;
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Orleans.Transactions.Tests
+namespace Orleans.Transactions.TestKit.xUnit
 {
-    public abstract class ConsistencyTransactionTestRunner : TransactionTestRunnerBase
+    public class ConsistencyTransactionTestRunnerxUnit : Base.ConsistencyTransactionTestRunner
```

Comment:
I don't think we need the Base. qualifier in the namespace.  Seems like the naming convention for Foo test of FooTestsRunner, FooTestRunnerXUnit, AzureFooTests Is suffice.  Base seems to imply some inheritance model, which we use, but don't have to.  Some runners in the system don't use inheritance.

Reply:
valid concern. Will remove.